src/index.js

```diff
@@ -100,6 +100,8 @@ import {
   DEFAULT_OPACITY_BY_DENSITY_FILL,
   DEFAULT_OPACITY_BY_DENSITY_DEBOUNCE_TIME,
   DEFAULT_GAMMA,
+  Z_NAMES,
+  W_NAMES,
```

Comment:
Small nit: Maybe an es6 `Set` could be useful for these values rather than an Array. I'll show some snippets below:

```javascript
Z_NAMES = new Set(Z_NAMES)
```

Reply:
I like that! Will make the changes accordingly